components/elm/src/biogeophys/SnowSnicarMod.F90

```diff
@@ -2169,18 +2329,39 @@ subroutine SNICAR_AD_RT (flg_snw_ice, bounds, num_nourbanc, filter_nourbanc,  &
              elseif(numrad_snw==5) then
                 ! Direct:
                 if (flg_slr_in == 1) then
-                   flx_wgt(1) = 1._r8
-                   flx_wgt(2) = 0.49352158521175_r8
-                   flx_wgt(3) = 0.18099494230665_r8
-                   flx_wgt(4) = 0.12094898498813_r8
-                   flx_wgt(5) = 0.20453448749347_r8
+                  if (atm_type_index == 0) then
```

Comment:
Change it to `atm_type_index == atm_type_default`

Reply:
Done